transports/file/src/test/resources/reliability/inbound-message-loss-flow.xml

```diff
@@ -30,7 +30,7 @@
         <flow-ref name="failingFlow"/>
     </flow>
     <flow name="failingFlow" processingStrategy="synchronous">
-        <test:component/>
+        <test:component throwException="true"/>
```

Comment:
It replaces programming introduction of exception via FunctionalTestComponent callback..